lib/index.js

```diff
@@ -99,7 +99,7 @@ function nodeNetCatSrc(port, input) {
 }
 
 const FUNCTION_PRIORITY = [nativeNC, nodeNC];
-const FLAGS = isElectron() ? ['--ms-enable-electron-run-as-node'] : [];
+const FLAGS = isElectron() && process.env.$syncRpcSpawnProcessFlags ? process.env.$syncRpcSpawnProcessFlags.split(',') : [];
```

Comment:
Btw using `.split(',')` is very uncommon - the standard way to convert between string and array is to use: `JSON.stringify([ your array ])` and `JSON.parse(...)`
That's how you're supposed to do it for say environment variables or creating new node process with arguments.

Reply:
The protocol is passing comma separated flags, personally I don't have preference. @tonescones Which way do you like better?

Reply:
Really? I just looked online but don't see anything official about what you said. If it really is official then leave as is.

Reply:
I think this is fine. let's just keep it simple for now